include/AT_NumericalRoutines.h

```diff
@@ -402,5 +403,13 @@ double AT_get_interpolated_y_from_interval( const double left_x,
 
 // TODO implement linear interpolation on logarithmic scale
 
+/**
+ * IDF of Landau distribution
+ * Adapted from gsl_ran_landau by explicitly specifying X variable (from uniform distribution [0,1])
+ * The code is based on CERNLIB G110 method (see http://hep.fi.infn.it/cernlib.pdf)
+ * @param[in] X - random number from uniform distribution [0,1]
+ * @return Landau random number
+ */
+double CL_ranlan_idf(const double X);
```

Comment:
In forjs branch I've used some file to mock Cernlibfuns, maybe here we should do the same and make some file that will convert gsl_ran_landau and provide funcion that will be used in EnergyLoss.c ?

Reply:
I mean new file to store function: 
- double CL_ranlan_idf(const double X);

Reply:
I will probably create one when implementing Vavilov distribution.